integration/testnet/network.go

```diff
@@ -533,19 +544,34 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 			// uncomment line below to point the access node exclusively to a single collection node
 			// nodeContainer.addFlag("static-collection-ingress-addr", "collection_1:9000")
 			nodeContainer.addFlag("collection-ingress-port", "9000")
-			nodeContainer.opts.HealthCheck = testingdock.HealthCheckCustom(healthcheckAccessGRPC(hostGRPCPort))
-			nodeContainer.Ports[AccessNodeAPIPort] = hostGRPCPort
-			nodeContainer.Ports[AccessNodeAPIProxyPort] = hostHTTPProxyPort
-			net.AccessPorts[AccessNodeAPIPort] = hostGRPCPort
-			net.AccessPorts[AccessNodeAPIProxyPort] = hostHTTPProxyPort
+
+			var accessNodeAPIPortKey, accessNodeAPIProxyPortKey string
+			if nodeConf.Unstaked {
+				accessNodeAPIPortKey = UnstakedAccessNodeAPIPort
+				accessNodeAPIProxyPortKey = UnstakedAccessNodeAPIProxyPort
+			} else {
+				nodeContainer.opts.HealthCheck = testingdock.HealthCheckCustom(healthcheckAccessGRPC(hostGRPCPort))
+				accessNodeAPIPortKey = AccessNodeAPIPort
+				accessNodeAPIProxyPortKey = AccessNodeAPIProxyPort
+			}
```

Comment:
As mentioned above, do we really need to distinguish between staked and unstaked for the sake of exposing Access API port?

From the point of view of a client, it should connect to the same port of an access node regardless of whether or not it is staked right?